bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly.properties

```diff
@@ -247,7 +247,7 @@ channel-type.shelly.temperature4.description = Temperature of external Sensor #4
 channel-type.shelly.temperature5.label = Temperature 5
 channel-type.shelly.temperature6.description = Temperature of external Sensor #5
 channel-type.shelly.targetTemp.label = Target Temperature
-channel-type.shelly.targetTemp.description = Target Temperature  in �C to be reached in auto-temperature mode
+channel-type.shelly.targetTemp.description = Target Temperature in C to be reached in auto-temperature mode
```

Comment:
Why not keeping °C ?

Reply:
is back

Reply:
Looks like it is not the right character.
![image](https://github.com/openhab/openhab-addons/assets/10186704/1112b5df-d1ed-4ed0-9382-8804a3f56b77)